index.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable */
+import { displayDate } from './modules/date.js';
+import {
+  openBookList,
+  openAddBook,
+  openContact
+} from './modules/navigation.js';
+
+import {
+  bookInfo,
+  title,
+  author,
+  form,
+  listBtn,
+  addBtn,
+  contactBtn
+} from './modules/variables.js';
+
+let bookArr = [];
+
+class Book {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
+
+  static bookDisplay() {
+    bookInfo.innerHTML = '';
+    for (let i = 0; i < bookArr.length; i += 1) {
+      const oneBook = document.createElement('div');
+      oneBook.className = 'book';
+      oneBook.innerHTML = `
+        <p class="title">"${bookArr[i].title}" by ${bookArr[i].author}</p>
+        <button class="remove" data-id=${bookArr[i].id}>Remove</button>
+      `;
+      bookInfo.appendChild(oneBook);
+      title.value = '';
+      author.value = '';
+    }
+    const removeBtn = document.querySelectorAll('.remove');
+    removeBtn.forEach((book) => {
+      book.addEventListener('click', () => {
+        const dataSet = parseInt(book.dataset.id, 10);
+        const btnId = bookArr.findIndex((object) => object.id === dataSet);
+        Book.removeBook(btnId);
+      });
+    });
+  }
```

Comment:
- [ ] it would be better to use the `es6` arrow function instead of `normal methods` in order to complete the requirements